src/components/ZclAttributeManager.vue

```diff
@@ -276,13 +279,33 @@ export default {
         return false
       }
     },
-    isDisabled(id, name, selectedClusterId, selectedCluster) {
```

Comment:
What about all the places where isDisabled is called from? Don't those need a signature change?

Reply:
no the new method isDisabledDefault is a new method for a specific use case

Reply:
basically, rather than changing is disabled by adding if and else statements I just created a new method

Reply:
Are there function calls to isDisabled anywhere else in the code?

Reply:
yes, whenever an attribute is disabled...I added comments

Reply:
That is not what I am asking. Is `isDisabled` function called from anywhere in this file?
You have changed the signature of the function definition from `isDisabled(id, name, selectedClusterId, selectedCluster)` to `isDisabled(id, selectedClusterId)` . 
Now, what about the same change everywhere else in the code where there are function calls to `isDisabled(id, name, selectedClusterId, selectedCluster)`. Don't those need to be changed as well to `isDisabled(id, selectedClusterId)` ?